File: LeetCode/2864/solution.cpp
// 2864. Maximum Odd Binary Number
class Solution {
public:
    string maximumOddBinaryNumber(string s) {
        int cnt1 = 0, cnt0 = 0;
        for(char ch: s){
            if(ch=='1') cnt1++;
            else cnt0++;
        }
        string ans = string(cnt1-1, '1') + string(cnt0, '0') + '1';
        return ans;
    }
};

File: LeetCode/2864/solution.java
// 2864. Maximum Odd Binary Number
class Solution {
    public String maximumOddBinaryNumber(String s) {
        int cnt1 = 0, cnt0 = 0;
        for(char ch: s.toCharArray()) {
            if(ch == '1') cnt1++;
            else cnt0++;
        }
        StringBuilder ans = new StringBuilder();
        ans.append("1".repeat(cnt1 - 1));
        ans.append("0".repeat(cnt0));
        ans.append("1");
        return ans.toString();
    }
}